src/components/inclusive-dates-calendar/inclusive-dates-calendar.tsx

```diff
@@ -788,22 +838,96 @@ export class InclusiveDatesCalendar {
               </div>
               {this.showKeyboardHint &&
                 !window.matchMedia("(pointer: coarse)").matches && (
-                  <button
-                    type="button"
-                    onClick={() => alert("Todo: Add Keyboard helper!")}
-                    class={this.getClassName("keyboard-hint")}
-                  >
-                    <svg
-                      xmlns="http://www.w3.org/2000/svg"
-                      height="1em"
-                      width="1em"
-                      viewBox="0 0 48 48"
-                      fill="currentColor"
-                    >
-                      <path d="M7 38q-1.2 0-2.1-.925Q4 36.15 4 35V13q0-1.2.9-2.1.9-.9 2.1-.9h34q1.2 0 2.1.9.9.9.9 2.1v22q0 1.15-.9 2.075Q42.2 38 41 38Zm0-3h34V13H7v22Zm8-3.25h18v-3H15Zm-4.85-6.25h3v-3h-3Zm6.2 0h3v-3h-3Zm6.15 0h3v-3h-3Zm6.2 0h3v-3h-3Zm6.15 0h3v-3h-3Zm-24.7-6.25h3v-3h-3Zm6.2 0h3v-3h-3Zm6.15 0h3v-3h-3Zm6.2 0h3v-3h-3Zm6.15 0h3v-3h-3ZM7 35V13v22Z" />
-                    </svg>
-                    {this.labels.keyboardHint}
-                  </button>
+                  <details class="{this.getClassName("keyboard-shortcuts")}">
```

Comment:
A syntax error here, it should be
`<details class={this.getClassName("keyboard-shortcuts")}>`
(without the singlequotes)